src/wagtail_localize_smartling/models.py

```diff
@@ -329,3 +329,7 @@ def get_or_create_from_source_and_translation_data(
         # If we get here we've got a proper background worker, so we can safely
         # enqueue the syncing of the job.
         background.enqueue(sync_job, args=(job.pk,), kwargs={})
+
+    @property
+    def status_label(self) -> str:
+        return JobStatus[self.status].label
```

Comment:
Issue: this looks like it's just reimplementing `Job.get_status_display()`

Reply:
ha, yes. I've simplified it